docker/start-app.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This configuration is a temporary workaround until we migrate to CCF 4.0.3, which supports
+# a cchost "config-timeout" option to accomplish a similar result. This logic is required when
+# the SCITT container is deployed in Kubernetes without any orchestration sidecars.
+
+WAIT_TIME_SEC=3
+TIMEOUT_SEC=5
+
+echo "Starting app"
+config_arg=${1:?first argument is the configuration argument to pass to cchost (e.g., "--config /path/to/config.json"))}
```

Comment:
Actually by looking into how the parameters passed it seems that his will not work, e.g.
https://github.com/microsoft/scitt-ccf-ledger/blob/main/docker/run-dev.sh#L93

it passes them as separate values to `docker run .... --config /path/to/config.json` so here $1 will be `--config` and $2 will be `/path/to/config.json`

Reply:
Good point. For some reason it was working fine when I tested in from the cluster(because I think it is passed a single string), but indeed it doesn't work when tested locally. I just pushed a fix to get all args.